lms/static/scripts/postmessage_json_rpc/server/test/index-test.js

```diff
@@ -0,0 +1,80 @@
+import { startRpcServer, getSidebarWindow, $imports } from '../index';
+
+describe('postmessage_json_rpc/server', () => {
+  let FakeServer;
+  let FakeRegister;
+
+  beforeEach(() => {
+    FakeRegister = sinon.stub();
+
+    FakeServer = sinon.stub().returns({
+      register: FakeRegister,
+      sidebarWindow: 'FakeSidebarWindow',
+    });
+
+    $imports.$mock({
+      './server': FakeServer,
+    });
+  });
+
+  afterEach('remove the server config from the document', () => {
```

Comment:
```suggestion
  afterEach(() => {
```

The first argument was obsolete here, as that is now handled in a separate `afterEach` below.